raw-data/UW/uw_inventories.tsv

```diff
@@ -0,0 +1,55 @@
+				e		e	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				e		ẽ	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				ɨ		ɨ	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				ɨ		ɨ̃	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				a	low central vowel	a	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				a	low central vowel	ã	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				u		u	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				u		ũ	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				o		o	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				o		õ	"Oral vowels can undergo nasalization… Nasality in this instance could be argued to be a suprasegmental feature that serves to  mark the plural of certain nouns."	"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
+				<ə>	only appears in past tense suffix /-tə/			
+				ĩ				"Vowels additionally are subject to a harmony process in the person prefixes and the negative suffix."
```

Comment:
I think that the vowel harmony system described in the source is best understood as *allomorphy* (where the phoneme in an affix changes to match phonemes in the root) rather than *allophony*.  So I'd be inclined to remove all the notes about the harmony system.  But feel free to push back and make a case for it, if you disagree.